examples/icestick/checker/Makefile

```diff
@@ -0,0 +1,48 @@
+# PROJ = example
+# PROJ = rs232demo
+PROJ = checker
+
+PIN_DEF = icestick.pcf
+DEVICE = hx1k
+
+all: $(PROJ).rpt $(PROJ).bin
+
+%.blif: %.v
+	yosys -p 'synth_ice40 -top top -blif $@' $<
+
+%.asc: $(PIN_DEF) %.blif
+	arachne-pnr -d $(subst hx,,$(subst lp,,$(DEVICE))) -o $@ -p $^
```

Comment:
Seems like a good time to switch these over to use nextpnr

Reply:
ok